apps/admin_api/priv/spec.yaml

```diff
@@ -3345,40 +3365,144 @@ components:
             expired_at: null
 
     ######################################
-    #           SETTING SCHEMAS          #
+    #       CONFIGURATION SCHEMAS        #
     ######################################
-    SettingResponse:
-      description: "The response schema for settings"
+    SettingSchema:
```

Comment:
Since we're using "configuration" now, should this be changed to `ConfigurationSchema`?

Also `SettingResponse` -> `ConfigurationResponse` a bit down below.

Reply:
I wish it was just settings, but it was taken, so instead we send back a configuration composed of a set of settings, that's why you see both settings and configuration in the spec depending on the level.